Automattic/jetpack
# Post categories: Add quick action to change the default category

Part of https://github.com/Automattic/wp-calypso/issues/95621
See pekYwv-4RI-p2

## Proposed changes:

Adds a new quick action to the WP Admin view of "Posts > Categories" so users can change the default category from there.

<img width="634" alt="Screenshot 2024-12-18 at 15 53 35" src="https://github.com/user-attachments/assets/66c2a065-e04a-44cb-b070-aa6ba9190e8c" />



### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Apply these changes to your WP.com site
- Go to `/edit-tags.php?taxonomy=category`
- Make sure there is a new "Set as default" quick action (except on the current default category)
- Make sure you can change the default category with this quick action
